Michael Bustillos- Objects

```diff
@@ -0,0 +1,49 @@
+/* Exercise 2: Objects Practice
+In this exercise, you will be practicing creating objects, object methods. */
+
+/* Object 1: Vending Machine Object
+Properties:
+Capacity -> have it start to be 100
+Stock -> have it start to be 50
+Methods:
+  addStock -> takes in the amount to stock, stocks if it doesn't go over capacity. Print to console if not possible.
+  getItem -> removes one item from the vending machine. Print to console if stock is 0. */
+  
+ let vendingMachine = {
+    capacity: 100;
+    stock: 50;
+    addStock: function ( store ) {
+        if ( store + stock > capacity ){
```

Comment:
Since you are accessing a member of vendingMachine, you need `this.stock` instead of `stock`. Same with capacity.